portfolio/src/main/webapp/script.js

```diff
@@ -76,12 +92,17 @@ function fetchComments(limit = undefined, username = undefined) {
     });
 }
 
-function getRequestParameter(limit, username) {
-  return 'limit=' + limit + (username != null ? '&username=' + username : '');
+function getRequestParameter(limit, username, order) {
+  return 'limit=' + limit 
```

Comment:
You seem to be using `undefined` everywhere but then this method checks for `null` instead? Is this a bug?

Reply:
I refactored the part for generating request parameter.